main.c

```diff
@@ -205,6 +224,19 @@ static bool init_emitter(FILE *ts_file)
 	if (!yaml_emitter_emit(&emitter, &event))
 		ERROR_GOTO(emitter_delete,
 				"Failed to emit YAML sequence mapping event");
+	/*
+	 * Filling header
+	 */
+	snprintf(data, MAX_SCALAR, "%" PRIu64, bench_ts_global->cores);
+	fill_map("cores", data);
+
+	snprintf(data, MAX_SCALAR, "%" PRIu64, bench_ts_global->freq);
+	fill_map("timer_frequency", data);
```

Comment:
Return with error on failure ?
The same comment could apply on calls to `fill_map()` from `fill_timestamp()`.

Reply:
@etienne-lms agree, will fix

Reply:
@etienne-lms fixed